src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte

```diff
@@ -0,0 +1,18 @@
+<div class="flex flex-col items-start justify-between md:flex-row">
+	<!-- TODO: Update colors with values from theme-variables when they are available there -->
```

Comment:
just as info, there is this PR about it:

- #3444

Reply:
👍